### PR TITLE
Added button to append items to list

### PR DESCRIPTION
**Pull request by Vinayswami91**

### CODE REVIEW

File: main.js [hunk 1]
```diff
@@ -1 +1,12 @@
 console.log(' hello . .this is cool');
+
+
```
Comment (amarbagineni) [inline main.js]: naming convention can be better, to have the purpose of it shown right away,  which will also save us from needing any comments in the code

Comment (amarbagineni) [reply main.js]: Example - 'but' => 'greetingButton'

Comment (Vinayswami91) [reply main.js]: Done.

Comment (amarbagineni) [reply main.js]: Did you check if the app is working ?

Comment (Vinayswami91) [reply main.js]: Fixed

**Pull request merged**